src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,14 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        User x = userService.findByEmail(email);
+        if (x == null) {
+            return false;
+        }
+        return x.getPassword().equals(password);
```

Comment:
1. bad `User` variable naming, never name variables like so except counters or lambdas
2. too complicated if-elfse logic. think about how to rewrite to do it in 1 line
```suggestion
        User user = userService.findByEmail(email);
        if (user == null) {
            return false;
        }
        return user .getPassword().equals(password);
```